stories/control/function.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { Meta } from '@storybook/react';
+
+import { Axis, Legend, Plot } from '../../src';
+import {
+  FunctionSeries,
+  FunctionSeriesProps,
+} from '../../src/components/FunctionSeries';
+import { DEFAULT_PLOT_CONFIG } from '../utils';
+
+export default {
+  title: 'API/FunctionSeries',
+  component: FunctionSeries,
+  args: {
+    max: 10,
+    min: 0,
+    step: 0.001,
+  },
+} as Meta<FunctionSeriesProps>;
+
+export function Control(args: FunctionSeriesProps) {
+  const getY = (x: number) => 4 * Math.sin(2 * x);
```

Comment:
Can you move this out of the Control function so it is not recreated on every render?

Also prefer using a function declaration instead of an arrow when possible:

```
function getY() {
 ...
}
```

Reply:
okey thank you